### PR TITLE
Add removeEmoji function to mail services

### PR DESCRIPTION
**Pull request by nicoaravena**
This PR fix an error ocurred when an item bought from the store goes with an emoji inside the message.

# Solution proposed
* Remove all emoji coming in the item message.

# How to test
* Go to https://getemoji.com/
* Buy some item from the store and add one emoji to the message

### CODE REVIEW

**Comment by Eronox** [comment]
Tested and works fine, however since we can assume everyone is using modern browsers and OS nowadays i'd vote for converting them to unicode to preserve them

**Comment by nicoaravena** [comment]
I don't know how the client manage the emojis, if works fine all this thing ingame and doesn't broke them, the I can do a new aproach

File: srv/wordpress/wp-content/plugins/acore-wp-plugin/src/Manager/Soap/MailService.php [hunk 1]
```diff
@@ -9,25 +9,30 @@ class MailService {
...
     // requires https://github.com/55Honey/Acore_SendAndBind
     public function sendItemAndBind($guid, $message, $itemId, $stack) {
-        $_message = addslashes($message);
+        $_message = addslashes(self::removeEmoji($message));
         $_itemId = intval($itemId);
         $_stack = intval($stack);
         return $this->executeCommand('.senditemandbind ' . $guid . ' ' . $_itemId . ' ' . $_stack . ' ' . $_message);
     }
+
+    public static function removeEmoji($text)
```
Comment (Helias) [inline srv/wordpress/wp-content/plugins/acore-wp-plugin/src/Manager/Soap/MailService.php:34]:
```suggestion
    public static function removeEmoji($text): string
```

**Comment by Helias** [comment]
I would like to merge this asap, because it's a bit urgent, if anyone read this can break the db, we could make the improvement in another PR

**Comment by nicoaravena** [comment]
@Helias just add your suggestion.

**Pull request merged**